CONTRIBUTING.md

```diff
@@ -76,6 +76,12 @@ If using `pip` and `virtualenv` (not recommended), you need to `cd` to the direc
 $ pip install -e .
 ```
 
+Finally, you need to install the subrepos in develop mode. This will install `spyder`, `spyder-kernels`, `qdarkstyle`, and `python-lsp-server` in develop mode.
```

Comment:
```suggestion
Finally, you need to install the core dependencies in the subrepos.
This will install `spyder`, `spyder-kernels`, `qdarkstyle`, and `python-lsp-server` in editable mode.
```

* Better, more precise/specific terminology.
* Avoid repetition
* Better phrasing
* OSPL